imports/lib/google-map.js

```diff
@@ -2,56 +2,96 @@ import React, { PropTypes } from 'react';
 import { Random } from 'meteor/random';
 import { createContainer } from 'meteor/react-meteor-data';
 import { Meteor } from 'meteor/meteor'
+import Marker from '../ui/marker'
```

Comment:
if you want to do this have to wrap export with `{[export]}`.

seems like this is not typically done for components https://github.com/meteor/simple-todos-react/blob/master/imports/ui/App.jsx#L8